src/raw.rs

```diff
@@ -240,7 +240,7 @@ pub fn hash_get(key: *mut RedisModuleKey, field: &str) -> *mut RedisModuleString
             REDISMODULE_HASH_CFIELDS as i32,
             CString::new(field).unwrap().as_ptr(),
             &res,
-            0,
+            ptr::null::<c_char>(),
```

Comment:
Why `c_char`? Should be `ptr::null::<RedisModuleString>()`, right?

Also: The same fix should be made to `HashSet` below.

Reply:
Well technically since we're passing in `REDISMODULE_HASH_CFIELD` the pointers are to C-strings. We could also make this a `ptr::null::<core::ffi::c_void>()`.

Reply:
I've pushed a rebase that applies the fix to `hash_set` as well.

Reply:
Ouch... this API is even uglier than I thought. You're right.